### ISSUE TITLE
Incompatible with Composer 2

### ISSUE THREAD

**Issue by lcharette** [userfrosting/UserFrosting#1117]
```
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - wikimedia/composer-merge-plugin dev-master requires composer-plugin-api ^1.1 -> found composer-plugin-api[2.0.0] but it does not match the constraint.
    - wikimedia/composer-merge-plugin[v1.4.0, ..., v1.4.1] require composer-plugin-api ^1.0 -> found composer-plugin-api[2.0.0] but it does not match the constraint.
    - wikimedia/composer-merge-plugin 1.5.x-dev is an alias of wikimedia/composer-merge-plugin dev-master and thus requires it to be installed too.
    - Root composer.json requires wikimedia/composer-merge-plugin ^1.4.0 -> satisfiable by wikimedia/composer-merge-plugin[v1.4.0, v1.4.1, 1.5.x-dev (alias of dev-master)].

You are using Composer 2, which some of your plugins seem to be incompatible with. Make sure you update your plugins or report a plugin-issue to ask them to support Composer 2.
```

This is dependent of an issue with [wikimedia/composer-merge-plugin](https://github.com/wikimedia/composer-merge-plugin), and can only be solved once https://github.com/wikimedia/composer-merge-plugin/pull/189#issuecomment-719514950 is merged and released. 

Other reference : https://github.com/wikimedia/composer-merge-plugin/issues/184

**Comment by lcharette** [comment]
We should be able to fix this now by upgrading `wikimedia/composer-merge-plugin`  to this new release : 
https://github.com/wikimedia/composer-merge-plugin/releases/tag/v2.0.1

**Comment by Silic0nS0ldier** [comment]
_And_ raising our minimum PHP version from 7.1 to 7.2 (as per plugin requirements). Otherwise looks to a fairly straightforward update thankfully.

**Comment by lcharette** [comment]
We're still supporting 7.1? Might as well release 4.5 while at it. Do you have any other change / BC that requires a major bump ?

**Comment by Silic0nS0ldier** [comment]
Yeah, NodeJS for ESM support. Not ready for prime time yet, but raising will let it release more easily.
Min is `^12.17.0 || >=13.2.0`.

**Comment by lcharette** [comment]
> Yeah, NodeJS for ESM support. Not ready for prime time yet, but raising will let it release more easily.
> Min is `^12.17.0 || >=13.2.0`.

Care to push the required change to develop and/or open PR? I'll see if I can release this fix over the weekend.

**Comment by Silic0nS0ldier** [comment]
Sure. I'll see about getting it up this afternoon (my time, so in around 9 hours).

**Comment by Silic0nS0ldier** [comment]
I've opened https://github.com/userfrosting/UserFrosting/pull/1138

**Comment by lcharette** [comment]
This issue is fixed in ab33733 on develop for 4.5.0 release.
